pkgs/top-level/python-packages.nix

```
@@ -9430,7 +9430,10 @@ in {
 
   tubeup = callPackage ../development/python-modules/tubeup { };
 
-  tumpa = callPackage ../development/python-modules/tumpa { };
+  tumpa = pkgs.libsForQt5.callPackage ../development/python-modules/tumpa {
+    inherit buildPythonPackage pythonOlder;
+    inherit (self) setuptools pyside2 johnnycanencrypt;
+  };
```

```suggestion
  tumpa = callPackage ../development/python-modules/tumpa {
    inherit (pkgs.libsForQt5) wrapQtAppsHook
  };
```
I think this should be more compatible for pythonPackages and reduce the amount of unexpected things happening.

---

Thanks.

This might be also interesting for @knedlsepp as I've borrowed it from `mayavi`.

---

Thx for pinging. I'll take a look if I might want to borrow back. :-)